rocky/rocky/templates/organizations/select_clearance_level_radio_input.html

```diff
@@ -0,0 +1,27 @@
+{% load i18n %}
+
+<fieldset>
+  <legend>
+    {{ field.label_tag }}
+  </legend>
+  {% for choice in field %}
+    <div class="horizontal-view">
+      <input type="radio"
+             name="{{ field.name }}"
+             value="{{ choice.data.value }}"
+             id="id_{{ field.name }}_{{ forloop.counter0 }}"
+             {% if choice.data.attrs.disabled %} disabled{% endif %}
+             {% if choice.data.attrs.checked %} checked{% endif %}/>
+      <label>
+        <ul class="level-indicator l{{ forloop.counter0 }}">
```

Comment:
I think you could use the `scan_level_indicator` partial for rendering the amount of paws after each radio button.